.github/workflows/land-blocking.yml

```diff
@@ -71,12 +71,23 @@ jobs:
         run: |
           export CTI_OUTPUT_LOG=$(mktemp)
           echo "::set-env name=CTI_OUTPUT_LOG::$CTI_OUTPUT_LOG"
-          JUMPHOST=${{secrets.CLUSTER_TEST_JUMPHOST}} \
+          if [[ ${{ secrets.ACTIONS_LBT_USE_K8S }} == true ]]; then
             ./scripts/cti \
-            --marker land \
-            --tag ${TEST_TAG} \
-            --report report.json \
-            --run bench
+              --k8s \
+              --tag ${TEST_TAG} \
+              --report report.json \
+              --run bench \
+              --k8s-fullnodes-per-validator=1 \
+              --k8s-num-validators=30 \
+              --emit-to-validator=false
```

Comment:
so bench actually requires `-- --is-fullnode` to be passed, emit-to-validator is ignored :(

Reply:
Done